scripts/report/01-make_reports.py

```diff
@@ -450,6 +451,15 @@ def run_report_average(session):
     evokeds = mne.read_evokeds(evoked_fname)
     fs_subjects_dir = config.get_fs_subjects_dir()
 
+    df_events = count_events(config.bids_root)
+    rep.add_htmls_to_section(
+        f'<div class="event-counts">\n'
```

Comment:
but now it's not centered anymore @hoechenberger :(

Reply:
How can we add custom CSS to the report?